mix.exs

```diff
@@ -2,8 +2,8 @@ defmodule NebulexRedisAdapter.MixProject do
   use Mix.Project
 
   @source_url "https://github.com/cabol/nebulex_redis_adapter"
-  @version "2.2.0"
-  @nbx_vsn "2.3.2"
+  @version "2.2.1"
+  @nbx_vsn "2.3"
```

Comment:
The issue here is that `@nbx_vsn` is used also when [compiling and running the test locally](https://github.com/cabol/nebulex_redis_adapter#testing) and it should point to a specific/latest tag, check it out: https://github.com/cabol/nebulex_redis_adapter/blob/master/mix.exs#L83 (and https://github.com/cabol/nebulex_redis_adapter/blob/master/mix.exs#L73). So, I'd suggest maybe creating another constant `@nbx_tag` for addressing this part, and then we can keep `@nbx_vsn` pointing to the Hex latest version (`2.4`). For example:

```elixir
@nbx_tag "2.4.1"
@nbx_vsn "2.4"
```

And then replace this section https://github.com/cabol/nebulex_redis_adapter/blob/master/mix.exs#L71-L77 by:

```elixir
defp nebulex_dep do
  if path = System.get_env("NEBULEX_PATH") do
    {:nebulex, "~> #{@nbx_tag}", path: path}
  else
    {:nebulex, "~> #{@nbx_vsn}"}
  end
end
```